### DISCUSSION TITLE
GraphQL Filtering Pull Request on CreatedDate

### DISCUSSION THREAD

**Discussion by rsarosh** [community/community#24611] [Apps, API and Webhooks]
<p>This query is working, but i am not able to find out how to filter it on Created Date. I want Pull Requests which are created today. </p>
<p>query GetPullRequest {<br>
 viewer {<br>
  name<br>
    organization(login: “MyOrgName”) {<br>
     name<br>
           repository(name: “My Repo Name”) {<br>
           name<br>
           pullRequests(last: 10, states: OPEN ) {<br>
           nodes {<br>
                   state<br>
                   title<br>
                   permalink<br>
                   createdAt<br>
                  author {<br>
                           login<br>
                           avatarUrl<br>
                 }<br>
             }<br>
          }<br>
       }<br>
}<br>
}</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>To my understanding, if the <code>PullRequestsConnection</code> doesn’t have a method of filtering by date, then it can’t be done via that type of query. However, you could use the <code>search</code> query like this:</p>
<pre><code>{
  search(query: "repo:atom/atom is:pr is:open created:&gt;2019-04-01", type: ISSUE, last: 100) {
    edges {
      node {
        ... on PullRequest {
          url
          title
          createdAt
        }
      }
    }
  }
}
</code></pre>
<p>Just replace <code>2019-04-01</code> with today’s date.</p>

**Comment by rsarosh** [comment]
<p>Thanks a lot <a class="mention" href="/lee-dohm">@lee-dohm</a> , where can i find more on the ‘search’, ‘query’ grammer. Any pointer will be appriciated.  I want to skip pr where author.login is equal to ‘greenkeeper’ etc.</p>

**Comment by lee-dohm** [comment]
<p>It’s the same query syntax as all of GitHub, so you can find the documentation in <a href="https://help.github.com/en/categories/searching-for-information-on-github" rel="nofollow noopener">our help docs on search</a>. So if you wanted to exclude greenkeeper, you could <a href="https://help.github.com/en/articles/searching-issues-and-pull-requests#search-by-author" rel="nofollow noopener">add <code>-author:app/greenkeeper</code> to your search query</a>.</p>
<p>I hope that helps!</p>

**Comment by letmejustputthishere** [comment]
<p>why is it that it doesn’t matter wether I use <code>first</code> or <code>last</code> in the <code>search</code> query? the order is always the same :thinking:</p>
